packages/blockchain-wallet-v4-frontend/src/components/NetworkWarning/NetworkWarning.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { IntlFormatters, useIntl } from 'react-intl'
+import { AlertCard } from '@blockchain-com/constellation'
+
+import { AlertCardWrapper } from './NetworkWarning.model'
+import { Variant, Warning } from './types'
+
+const getSendProps = (warning: Warning, formatMessage: IntlFormatters['formatMessage']) => ({
+  content: formatMessage(
+    {
+      defaultMessage:
+        'Please send {symbol} on the {network} Network for this wallet only. We cannot recover lost funds.',
+      id: 'networkWarning.send.content'
+    },
+    warning
+  ),
+  title: formatMessage(
+    {
+      defaultMessage: 'Only send {symbol} on the {network} network',
+      id: 'networkWarning.send.title'
+    },
+    warning
+  )
+})
+
+const getReceiveProps = (warning: Warning, formatMessage: IntlFormatters['formatMessage']) => ({
+  content: formatMessage(
+    {
+      defaultMessage:
+        'Please receive {symbol} on the {network} Network for this address only. We cannot recover lost funds.',
+      id: 'networkWarning.receive.content'
+    },
+    warning
+  ),
+  title: formatMessage(
+    {
+      defaultMessage: '{symbol} is on the {network} network',
+      id: 'networkWarning.receive.title'
+    },
+    warning
+  )
+})
+
+export type Props = {
+  variant: Variant
+  warning: Warning | null
```

Comment:
`null` is used to denominate absence of the warning in other parts of the feature too (e.g. in `getCoinNetworkWarning`). Personally, I think `null` has a better semantic meaning than `undefined`. Happy to discuss if you think otherwise.